README.md

```diff
@@ -127,18 +131,14 @@ an updated model state with a cmd for the store to process. A cmd is just
 an object that describes what side effect should run, and what to do with
 the result. There are several options for cmds, all available under the `Cmd` object:
 
-- `call(functionToCall, resultActionCreator, ...args)`
-  - Accepts a function to call that returns a value when called with `args`, and an action creator to transform the result into an action that gets dispatched.
-- `promise(functionToCall, resolveActionCreator, rejectActionCreator, ...args)`
-  - Accepts a function to call that returns a `Promise` called with `args`, and action creators that transform the promise resolution or rejection value respectively into 
-- `constant(action)`
-  - Accepts an `action` instance to dispatch immediately once the current dispatch cycle is completed.
+- `run(functionToCall, options)`
```

Comment:
it's in the api docs page. wasn't sure if i needed to go into too much detail here since the examples are pretty clear. i can though